ice40/utils/ice40_import_bel_timing.py

```diff
@@ -0,0 +1,316 @@
+"""Import timing information from icestorm generated SDF file and updates
+timing in the architecture file.
+"""
+
+import argparse
+from sdf_timing.sdfparse import parse as sdf_parse
+import lxml.etree as ET
+import re
+import logging
+from collections import namedtuple
+
+
+class pin_map(namedtuple('pin_map', 'sdf arch is_clk')):
```

Comment:
`pin_map` isn't the normal formatting for a class? `PinMap` would be the normal name?

Reply:
Done